src/main/java/com/github/lunatrius/schematica/proxy/ClientProxy.java

```diff
@@ -257,19 +264,20 @@ public static ImmutablePair<Boolean, ImmutableTriple<Integer, Integer, Integer>>
                     Map<String, Integer> coordinateMap = schematicMap.get(schematicName);
                     if (coordinateMap.containsKey("X") && coordinateMap.containsKey("Y")
                             && coordinateMap.containsKey("Z")) {
-                        return new ImmutablePair<>(
+                        return new ImmutableTriple<>(
                                 true,
+                                coordinateMap.get("Rotation"),
```

Comment:
Will this work if you load an old file that didn't have the Rotation field?

Reply:
It does work in my testing, though it does print an error in the logs, please replace with
```java
.getOrDefault("Rotation",0)
```

Reply:
Don't know how i didn't notice that. 
Thanks for spotting it :)